app/ui/components/footer.tsx

```diff
@@ -1,9 +1,3 @@
-import { Box } from '@material-ui/core';
-
 export default function Footer() {
-  return (
-    <Box display="flex" bgcolor="grey.200" p={2} alignItems="center">
-      Footer
-    </Box>
-  );
+  return <div>Footer</div>;
```

Comment:
If this is in another task I think you shouldn't modify this file for on this branch

Reply:
the footer with the Box element was an error because in the import it was a material ui component but now we use chakra component but I'm going to put the Box of chakra insted of div